package.json

```diff
@@ -136,7 +138,6 @@
     "typescript": "^4.8.4",
     "webpack": "^5.77.0",
     "webpack-cli": "^4.9.1",
-    "webpack-dev-server": "^4.7.3",
-    "circular-dependency-plugin": "^5.2.2"
+    "webpack-dev-server": "^4.7.3"
   }
 }
```

Comment:
(General comment)
Expand the documentation on the bar chart (`bar-chart.adoc`) to incorporate these new changes.